docs/debug-api.md

```diff
@@ -1,12 +1,71 @@
 # Debug APIs
 
-**NOTE:** This document describes APIs that provide support to external debugging tools. Do not use these APIs in application code.  For info and help on debugging promises in your application, see [Debugging Promises](api.md#debugging-promises).
+**NOTE:** This document describes APIs that provide support to external debugging tools. Do not use these APIs in normal application logic.  For info and help on debugging promises in your application, see [Debugging Promises](api.md#debugging-promises).
 
 These APIs allow debugging tools to receive information about promise errors.  For example, when.js's default [builtin unhandled rejection reporting](#debugging-promises) is built using these APIs, as is the long async stack trace support in [`when/monitor/console`](#whenmonitorconsole).
 
 A debugger could use them to do any number of things, such as display a list of "currently unhandled promise rejections", or send error reports to an error aggregation service.
 
-## Examples
+## Global rejection events
+
+when.js &lt;= 3.7.0 supports global `window` events in browsers, and `process` events in Node, similar to Node's `'uncaughtException'` event. This allows applications to register a handler to receive events from all promise implementations that support these global process events.
+
+The events are:
+
+* `'unhandledRejection'`: fired when an unhandled rejection is detected
+* `'rejectionHandled'`: fired when rejection previously reported via an '`unhandledRejection'` event becomes handled
+
+## Browser window events
+
+The following example shows how to use global `process` events in Node.js to implement simple debug output:
```

Comment:
copy-n-paste engineering #fail! :)

Perhaps you meant this?

"The following example shows how to use global `window` events in browser environments to implement simple debug output:"